bin/reth/src/commands/bitfinity_import.rs

```diff
@@ -164,32 +167,106 @@ impl BitfinityImportCommand {
         );
 
         // override the tip
-        let tip = if let Some(tip) = remote_client.tip() {
-            tip
+        let safe_block = if let Some(safe_block) = remote_client.safe_block() {
```

Comment:
@itsyaasir @Maximkaaa This seems to be wrong. If there are no safe blocks the import is skipped, but then the code to confirm a block is never executed so there will never be safe blocks

Reply:
Yes, I missed the case when the safe block is not among the newly loaded block. Fixed it.